app/models/user.rb

```diff
@@ -9,4 +9,12 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  has_many :friendships,foreign_key: 'send_friend_id'
+  has_many :send_friends,through: :friendships
+
+  has_many :friendships,foreign_key: 'recieved_friend_id'
+  has_many :recieved_friends,through: :friendships
+
+
 end
```

Comment:
Add tests for the usage of options `dependent`, `foreign_key` and `through`.

Example line 10, your test should be, ` it { should have_many(:comments).class_name('Comment').dependent(:destroy)}`.